packages/templates/typescript/src/documents.handlebars

```diff
@@ -3,27 +3,24 @@
 {{#unless @root.config.noNamespaces}}
 export namespace {{ toPascalCase name }} {
 {{/unless}}
-  export type {{#if @root.config.noNamespaces}}{{ toPascalCase name }}{{/if}}Variables = {
-  {{#each variables}}
-    {{#if @root.config.immutableTypes }}readonly {{/if}}{{ name }}{{ getOptionals this }}: {{ convertedType this }};
-  {{/each}}
-  }
+export type {{#if @root.config.noNamespaces}}{{ toPascalCase name }}{{/if}}Variables = {
```

Comment:
Can you please fix it? your IDE changed the indentation :X

Reply:
Sorry, I should turn off the auto-formatting of VSCode. ._.